packages/client-core/src/user/menus/ProfileMenu.tsx

```diff
@@ -441,11 +441,20 @@ const ProfileMenu = ({ hideLogin, onClose }: Props): JSX.Element => {
       <div className="mt-5 grid w-full grid-cols-1 gap-y-4">
         {isGuest && !originallyAcceptedTOS && (
           <>
-            <Checkbox
-              checked={checkedTOS.value}
-              onChange={() => checkedTOS.set((v) => !v)}
-              label={t('user:usermenu.profile.agreeTOS')}
-            />
+            <div className="flex w-full items-center justify-start gap-x-1">
+              <Checkbox
+                checked={checkedTOS.value}
+                onChange={() => checkedTOS.set((v) => !v)}
+                label={t('user:usermenu.profile.agreeTOS')}
+              />
+              <a
+                className="inline text-sm text-text-primary underline-offset-4 hover:text-ui-hover-primary hover:underline"
+                href="https://www.ir.world/terms-of-service"
+                target="_blank"
+              >
+                {t('user:usermenu.profile.termsOfService')}
```

Comment:
instead of the static link, we can add a the [client-settings termsOfService](https://github.com/ir-engine/ir-engine/pull/1073/files#diff-c64aa0e191e235a4784e295e05dae49b7529168a3ed3d45f35549b9a974eef02R73)